index.html

```diff
@@ -13,10 +13,36 @@
     />
   </head>
   <body>
-    <h1>Todos</h1>
-    <header>
-      <input class="inp" id="myinp" type="text" placeholder="wpisz coś" />
-    </header>
+    <div class="container">
+      <h1>Todos <img src="icon.png" alt="todoicon" /></h1>
+      <header>
+        <input class="inp" id="myinp" type="text" placeholder="wpisz coś" />
```

Comment:
Nie używamy raczej id. Odnosimy się po klasie.
Jeśli jest to jedyny input w aplikacji nie musisz dodawać klasy
Dodatkowo nazwa klasy powinna być bardziej klarowna np. addInput z notacją camel case